x-pack/plugins/reporting/export_types/common/layouts/print_layout.ts

```diff
@@ -14,12 +14,13 @@ interface BrowserClient {
 }
 
 export class PrintLayout extends Layout {
-  public selectors = {
+  public readonly selectors = {
     screenshot: '[data-shared-item]',
     renderComplete: '[data-shared-item]',
     itemsCountAttribute: 'data-shared-items-count',
     timefilterFromAttribute: 'data-shared-timefilter-from',
     timefilterToAttribute: 'data-shared-timefilter-to',
+    toastHeader: '[data-test-subj="euiToastHeader"]',
```

Comment:
Small nitpick: seems odd to mix selectors with dom-attributes in a selectors object. Not relevant to your PR but had me scratching my head for a second since some values had brackets and some didn't.

Reply:
I agree, it seems a little weird that it was originally done this way.